ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml

```diff
@@ -196,7 +196,7 @@ tests:
   interval: 48h
```

Comment:
> - as: e2e-metal-ipi-sdn-serial-virtualmedia
I wonder should we also rename this to "e2e-metal-ipi-sdn-serial-virtualmedia-bond"
so its more obvious to the casual observer ?

Reply:
Yeah. Probably makes sense too. Let me update it...

Reply:
Updated it to include the suffix